debian/control

```diff
@@ -1,7 +1,7 @@
 Package: wireguard
 Source: wireguard
 Version: WIREGUARD_VER-1
```

Comment:
You'll probably want to make that -1 a placeholder too, right? So that you can release fixes like this?